pkg/virt-controller/watch/drain/evacuation/evacuation_test.go

```diff
@@ -296,6 +296,39 @@ var _ = Describe("Evacuation", func() {
 			controller.Execute()
 		})
 
+		It("Shound do nothing when active migrations exceed the configured concurrent maximum", func() {
+			maxParallelMigrationsPerCluster := uint32(2)
+			maxParallelMigrationsPerSourceNode := uint32(1)
```

Comment:
@jcanocan Can you please make them const? `const maxParallelMigrationsPerCluster = uint32(2)`